src/openvic-simulation/military/Wargoal.hpp

```diff
@@ -0,0 +1,127 @@
+#pragma once
+
+#include "openvic-simulation/types/IdentifierRegistry.hpp"
+#include "openvic-simulation/types/EnumBitfield.hpp"
+#include "openvic-simulation/Modifier.hpp"
+
+namespace OpenVic {
+    struct WargoalTypeManager;
+
+    enum class peace_options_t : uint16_t {
+        PO_ANNEX =                 0b1000000000000000,
+        PO_DEMAND_STATE =          0b0100000000000000,
+        PO_COLONY =                0b0010000000000000,
+        PO_ADD_TO_SPHERE =         0b0001000000000000,
+        PO_DISARMAMENT =           0b0000100000000000,
+        PO_REPARATIONS =           0b0000010000000000,
+        PO_REPAY_DEBT =            0b0000001000000000,
+        PO_REMOVE_PRESTIGE =       0b0000000100000000,
+        PO_MAKE_PUPPET =           0b0000000010000000,
+        PO_RELEASE_PUPPET =        0b0000000001000000,
+        PO_STATUS_QUO =            0b0000000000100000,
+        PO_INSTALL_COMMUNISM =     0b0000000000010000,
+        PO_REMOVE_COMMUNISM =      0b0000000000001000,
+        PO_REMOVE_CORES =          0b0000000000000100, // only usable with ANNEX, DEMAND_STATE, or TRANSFER_PROVINCES
+        PO_TRANSFER_PROVINCES =    0b0000000000000010,
+        PO_CLEAR_UNION_SPHERE =    0b0000000000000001
```

Comment:
looks like you are missing dismantle forts from this list

Reply:
also, technically po_destroy_naval_bases, although I don't know if a mod has ever used that without also destroy forts

Reply:
Yeah that's where PO_DISARMAMENT comes from, I'll test sometime if they can be used discretely

Reply:
po_disarmament is its own thing. It appears in the cut down to size cb in the base game. I am talking about
```
	po_destroy_forts = yes
	po_destroy_naval_bases = yes
```
which can be found in the base game's dismantle forts cb